### ISSUE TITLE
Microsoft Graph - Turn on-off Still instructs "Go to Classic site collections page"

### ISSUE THREAD

**Issue by Toth-Les** [MicrosoftDocs/OfficeDocs-SharePoint#2789]
Office 365 Sharepoint Sites page still pops up a banner indicating:
"You're seeing a limited version of this page because Office Graph is turned off. Learn more about turning on Office Graph."  With a link that brings you to:
https://support.office.com/en-US/client/results?Shownav=true&lcid=1033&ns=SPOSTANDARD&version=16&omkt=en-US&ver=16&mode=Touch&helpId=WSSEndUser_SPHomeNoGraphBanner

This Site then directs you to "If you are the SharePoint admin, see Delve for Microsoft 365 admins" - also with a very convenient link that brings you to https://docs.microsoft.com/en-us/sharepoint/delve-for-office-365-admins#BKMK_DelveOnOff
The instructions found here tell you quite clearly to:

1. - Go to the Settings page of the new SharePoint admin center, and sign in with an account that has admin permissions for your organization.
2. - At the bottom of the page, **_select classic settings page._**
3. - Under Delve (powered by Office Graph), select one of the following:
4. - Enable Delve and related features
5. - Disable Delve and related features

The Problem is that the "Classic Settings Page" no longer exists and has been deprecated.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b21ad738-c2a1-ef1f-d745-eebe907311ac
* Version Independent ID: e931c7e1-fdb4-414b-986d-d7755fc797c4
* Content: [Delve for admins - SharePoint in Microsoft 365](https://docs.microsoft.com/en-us/sharepoint/delve-for-office-365-admins#BKMK_DelveOnOff)
* Content Source: [SharePoint/SharePointOnline/delve-for-office-365-admins.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointOnline/delve-for-office-365-admins.md)
* Service: **sharepoint-online**
* GitHub Login: @gretel
* Microsoft Alias: **gretel**

**Comment by scanum** [comment]
@Toth-Les Thank you for submitting feedback and contributing to the docs. We are currently investigating this

**Comment by get-itips** [comment]
Hello @Toth-Les 
How are you? Hope you are doing good 👍 
I followed the instructions successfully in my test tenant:

Go to https://admin.microsoft.com/sharepoint?page=settings&modern=true

At the bottom of the page, select classic settings page.


![image](https://user-images.githubusercontent.com/36666927/122617954-7e4c0280-d04a-11eb-871b-f8b8bbadc484.png)

Under Delve (powered by Office Graph), select one of the following:

Enable Delve and related features

Disable Delve and related features

![image](https://user-images.githubusercontent.com/36666927/122617983-90c63c00-d04a-11eb-9c92-3f5e734fe6b1.png)


(maybe this text differs a little and the article can be updated)


please correct me if I misunderstood something.

Hope this information helps you

**Comment by Toth-Les** [comment]
Yes – the link you have circled “classic settings page” is the one that wasn’t working.
That link is working now – thank you.

Having that link functional, I was able to enable Delve and once I refreshed my Sharepoint Sites page, the banner has gone away.

Since that link was not functional, I’d assumed that they had deprecated the “Classic Settings Page” prematurely.  That may be what happened, but it has been put back now, so all is good.

***@***.***D76451.C53CBE60]

Les Bergsman  |  I.T. Director
220 S. River St.  Seattle, WA 98108  |  C: 206-730-1450  |  tothconstruction.com
